src/Entity/Field/MarkdownField.php

```diff
@@ -5,11 +5,18 @@
 namespace Bolt\Entity\Field;
 
 use Bolt\Entity\Field;
+use Bolt\Utils\Markdown;
 use Doctrine\ORM\Mapping as ORM;
 
 /**
  * @ORM\Entity
  */
 class MarkdownField extends Field implements Excerptable
 {
+    public function __toString(): string
+    {
+        $markdown = new Markdown();
+
+        return $markdown->toHtml(implode(', ', $this->getValue()));
```

Comment:
how can markdown have more that 1 item in value array?

Reply:
Future-proofing. The value of _each_ field is basically an asociative array. Some fields (like 'embed' or 'image' have more items. Apart from simply treating them all the same this way, it allows us later to perhaps add the compiled HTML for performance, or a 'flavor' value for normal or GFM markdown.

Reply:
Thinking about this some more.. The `implode` is ugly.. I'll change it to `reset($value)`.